source/disassemble.cpp

```diff
@@ -182,6 +211,12 @@ spv_result_t Disassembler::HandleInstruction(
     EmitOperand(inst, i);
   }
 
+  if (comment_ && opcode == SpvOpName) {
```

Comment:
Not sure I get the reasoning behind this. The id used in the disassembly is already in the instruction. If you really want the numeric id, maybe this should be disabled if outputting raw ids.

Reply:
I thought about that, it's  a larger change as the disassembly code doesn't know we're outputting raw ids at this point. The namer handles that and is injected into the disassember.

I think the default is to output named id's, so the majority of the time this makes sense. Wasn't sure if the added complexity was worth it.

Reply:
This seems fine to me.
Yes, the default is to use the specified name as the Id's name.